fastembed/image/transform/functional.py

```diff
@@ -122,3 +122,18 @@ def pil2ndarray(image: Union[Image.Image, np.ndarray]):
     if isinstance(image, Image.Image):
         return np.asarray(image).transpose((2, 0, 1))
     return image
+
+
+def resize2square(
+    image: Image.Image,
+    size: int,
+    fill_color: Optional[Union[str, int, tuple[int, ...]]] = None,
+    resample: Union[Image.Resampling, int] = Image.Resampling.BICUBIC,
+) -> Image.Image:
+    resized_image = resize(image=image, size=size, resample=resample)
+
+    new_image = Image.new(mode="RGB", size=(size, size), color=fill_color)
+    left = (size - resized_image.size[0]) // 2
+    top = (size - resized_image.size[1]) // 2
+    new_image.paste(resized_image, (left, top))
+    return new_image
```

Comment:
we already have `resize`, let's not introduce new functions, we can add an additional parameter like `preserve_aspect_ratio` or something like this to `resize`.
If it's false then we should just resize image to the required size (preserving aspect ratio is useful when later we have crop)